ui/packages/catalog-ui-search/src/main/webapp/react-component/select-coordinates/select-coordinates.tsx

```diff
@@ -149,7 +131,7 @@ const generateClipboardHandler = (
     })
     clipboardInstance.on('error', (e: any) => {
       announcement.announce({
-        title: 'Press Ctrl+C to copy',
+        title: 'Could not copy distance to clipboard',
```

Comment:
✏️ Can you add something here to notify the user that they can still use `Ctrl+C` to manually copy it?